augur/routes/pull_request_reports.py

```diff
@@ -421,14 +421,37 @@ def filter_data(df, needed_columns, not_null_columns=[]):
             print("Developer error, not null columns should be a subset of needed columns")
             return df
 
-    @server.app.route('/{}/pull_request_reports/average_commits_per_PR/'.format(server.api_version), methods=["GET"])
-    def average_commits_per_PR():
+    def get_repo_id_start_date_and_end_date():
+
+        """ Gets the repo_id, start_date, and end_date from the GET requests array
+
+        :return: repo_id - id of the repo data is being retrieved for
+        :return: start_date - earliest time on visualization. Defaults to the January 1st of last year
+        :return: end_date - latest time on visualization. Defaults to current date
+        """
 
         now = datetime.datetime.now()
 
-        repo_id = int(request.args.get('repo_id'))
```

Comment:
Same question about int casting. I'm just asking to make sure I understand. My guess is you have tested it, and it is not necessary. So, then no problem.

Reply:
Yes, it is casted in the return statement, after I've confirmed that it isn't a None type, because trying to cast a None type to int will return an error, and give the user a 500 server error